src/NuGet.Core/NuGet.CommandLine.XPlat/Strings.resx

```diff
@@ -712,4 +712,14 @@ For more information, visit https://docs.nuget.org/docs/reference/command-line-r
   <data name="AddClientCertCommandDescription" xml:space="preserve">
     <value>Adds a client certificate configuration that matches the given package source name.</value>
   </data>
-</root>
+  <data name="AddPkg_PackagePrerelease" xml:space="preserve">
+    <value>Consider prerelease packages when determining the latest version to install.</value>
+  </data>
+  <data name="Error_PrereleaseWhenVersionSpecified" xml:space="preserve">
+    <value>The --prerelease and --version options are not supported in the same command.</value>
+  </data>
+  <data name="Error_NoVersionsAvailable" xml:space="preserve">
+    <value>There are no stable versions available for the package '{0}'.</value>
```

Comment:
Isn't this also being shown when using --prerelease ? Therefore, "stable" isn't always right.